pkg/clustermanager/cluster_manager.go

```diff
@@ -137,6 +139,18 @@ func New(clusterClient ClusterClient, networking Networking, writer filewriter.F
 	return c
 }
 
+func GetMaxWaitPerMachine() time.Duration {
```

Comment:
This is a good idea!

However I would like to see this being read at the beginning of the program (from the cmd package) and then being injected in the cluster manager constrcutor

We can probably use the `CliConfig`, pass it to the dependency factory and then inject this value from there into `ClusterManager`


Happy to to move this conversation offline if that's easier :)